src/demo.ts

```diff
@@ -584,6 +601,8 @@ export class DemoFile extends EventEmitter {
   private _parseRecurse() {
     this._recurse();
 
+    if (this.paused) return;
```

Comment:
This is going to trigger `_parseRecurse` to be called constantly even though it's not doing anything. I think it would be better to cancel the timer instead on `pause`, then to restart the timer on `resume`.

Reply:
Thanks! I'll reconsider this